packages/react-scripts/config/webpack.config.js

```diff
@@ -629,6 +629,7 @@ module.exports = function(webpackEnv) {
             isolatedModules: true,
             noEmit: true,
             jsx: 'preserve',
+            baseUrl: config.baseUrl,
```

Comment:
We probably can leave this until we do the paths PR, but i think we should start using the app's `tsconfig.json` instead of hard-coding the values here. This would support `paths` and `baseUrl` implicitly.

Reply:
Why is this required? I'm not sure I understand.

Reply:
In my first iteration, I was only showing an error when the `baseUrl` didn't match our requirements and "ignore" it for the webpack configuration. Now we throw an actual error and prevent the rest of the build process we no longer have to overwrite it here.

Reply:
Ah, OK. We can probably remove this then. 😄